lightning/src/ln/channel.rs

```diff
@@ -1623,11 +1622,17 @@ pub(super) struct FundingScope {
 	next_local_commitment_tx_fee_info_cached: Mutex<Option<CommitmentTxInfoCached>>,
 	#[cfg(any(test, fuzzing))]
 	next_remote_commitment_tx_fee_info_cached: Mutex<Option<CommitmentTxInfoCached>>,
+
+	pub(super) channel_transaction_parameters: ChannelTransactionParameters,
+
+	/// The transaction which funds this channel. Note that for manually-funded channels (i.e.,
+	/// [`ChannelContext::is_manual_broadcast`] is true) this will be a dummy empty transaction.
+	funding_transaction: Option<Transaction>,
```

Comment:
would it be possible to have just the transaction ID here?  it's not so good to carry around the funding transaction for every signing operation in memory limited environments.

Reply:
ah, of course, thank you